src/js/containers/ImportAccount/ImportByWalletConnect.js

```diff
@@ -42,23 +43,52 @@ export default class ImportByWallletConnect extends React.Component {
     async connect(e) {
         var wallet = this.state.wallet
         try {
-            var address = await wallet.getAddress()
-            this.setState({                
+            var {address, chainId} = await wallet.getAddress()
```

Comment:
Please dont add chainId in here, seperate 2 function: getAddress(), getId(). Check getId() first before proceed with address

Reply:
I already separated them into 2 functions